From 1dc8873b416bd784eee1d2533eb5d03e77374654 Mon Sep 17 00:00:00 2001
From: Dan McDonald <danmcd@joyent.com>
Date: Wed, 1 Aug 2018 14:59:31 -0400
Subject: [PATCH] OS-7009 varpd fails to start on boot, missing overlay module
 Reviewed by: Jason King <jbk@joyent.com> Reviewed by: John Levon
 <john.levon@joyent.com>

---
 usr/src/cmd/zoneadm/zones.xml | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/usr/src/cmd/zoneadm/zones.xml b/usr/src/cmd/zoneadm/zones.xml
index 1b6f8cd49b..b094bc660b 100644
--- a/usr/src/cmd/zoneadm/zones.xml
+++ b/usr/src/cmd/zoneadm/zones.xml
@@ -54,6 +54,19 @@
 		<service_fmri value='svc:/milestone/multi-user-server' />
 	</dependency>
 
+	<!--
+	    Until overlay device creation is moved out of the zone
+	    state-change script, zones must be dependent on varpd's
+	    successful launch.
+	-->
+	<dependency
+		name='varpd'
+		type='service'
+		grouping='require_all'
+		restart_on='none'>
+		<service_fmri value='svc:/network/varpd' />
+	</dependency>
+
 	<dependency
 		name='metadata'
 		type='service'
-- 
2.21.0

